astropy/modeling/tests/test_models.py

```diff
@@ -729,6 +729,17 @@ def test_tabular_interp_1d():
     assert_quantity_allclose(model(xnew.to(u.nm)), ans1)
     assert model.bounding_box == (0 * u.nm, 4 * u.nm)
 
+    # Test with no units on points.
+    model = LookupTable(points=points, lookup_table=values * u.nJy)
+    assert_quantity_allclose(model(xnew), ans1)
```

Comment:
Your example in the issue uses a single-element array to demonstrate the bug. Do we not need a single-element array in test here?